docs/configuration/README.md

```diff
@@ -644,6 +644,8 @@ to wait before saving them to the backing store.
 [cache_lookups_older_than: <duration>]
 
 # Limit how long back data can be queries. Default is disabled.
+# If you configure `table_manager_config.retention_period` > 0
+# then you can set the same value here.
```

Comment:
I would suggest changing it to:
This should always be set to a value less than or equal to what is set in `table_manager_config.retention_period`.

Can you also please add a note to https://github.com/grafana/loki/blob/ae0aa811018a1371577c00dc1a9d49796c3b2528/docs/operations/storage/retention.md#loki-storage-retention